source/vistas/ui/controls/gl_canvas.py

```diff
@@ -8,6 +8,10 @@
 
 
 class GLCanvas(wx.glcanvas.GLCanvas):
+    """
+    Main panel for 3D rendering. A shared OpenGL context is created when the first instance of this class is used.
```

Comment:
It's unclear what "main panel" means here. We use the terms `main_viewer_panel` and `main_panel` elsewhere and this is related to neither.